backends/simulated.go

```diff
@@ -167,10 +180,29 @@ func (b *SimulatedBackend) StorageAt(ctx context.Context, contract common.Addres
 
 // TransactionReceipt returns the receipt of a transaction.
 func (b *SimulatedBackend) TransactionReceipt(ctx context.Context, txHash common.Hash) (*types.Receipt, error) {
-	receipt, _, _, _ := rawdb.ReadReceipt(b.database, txHash)
+	receipt, _, _, _ := rawdb.ReadReceipt(b.database, txHash, b.config)
 	return receipt, nil
 }
 
+// TransactionByHash checks the pool of pending transactions in addition to the
+// blockchain. The isPending return value indicates whether the transaction has been
```

Comment:
The documentation here refers to `isPending`, but there's no such reference to that variable for the returned bool value — perhaps update the documentation to match? Something like: "The returned bool indicates" perhaps?

Reply:
all of this comes from the geth source, not going to change it ...